components/02-elements/tab/Tab.vue

```diff
@@ -0,0 +1,234 @@
+<template>
+  <div
+    :class="['tab', modifier && 'tab--' + modifier]"
+  >
+    <template
+      v-for="tab in tabs"
+    >
+      <button
+        :id="tab.tabId"
+        :key="'tab' + tab.tabId"
+        :class="[
+          'tab__title',
+          tab.tabId === activeTab && 'tab__title--active'
+        ]"
+        :data-tab="tab.tabId"
+        :aria-controls="tab.tabId"
+        :aria-selected="tab.tabId === activeTab ? 'true' : 'false'"
```

Comment:
IMO you can reduce it to
```suggestion
        :aria-selected="tab.tabId === activeTab"
```